### PR TITLE
Add IdVP feature documentation

### PR DESCRIPTION
**Pull request by JayaShakthi97**
## Purpose
IdVP feature UX is revamped and enabling by default. This PR adds the IS documentation for the feature.

https://github.com/user-attachments/assets/1b31fae4-3251-4ad3-b9d9-0a5cf6e3b33a

https://github.com/user-attachments/assets/af61dc60-d8a1-4550-8495-8211715c0c75

### CODE REVIEW

File: en/identity-server/next/docs/apis/restapis/identity-verification-providers.yaml [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+    BasicAuth:
+      type: http
+      scheme: basic
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://localhost:9443/oauth2/authorize'
+          tokenUrl: 'https://localhost:9443/oauth2/token'
+          scopes: {}
```
Comment (pavinduLakshan) [inline en/identity-server/next/docs/apis/restapis/identity-verification-providers.yaml:515]:
```suggestion
          scopes: {}

```

File: en/includes/guides/identity-verification/manage-identity-verification-provider.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Edit an Identity Verification Provider
+
+An administrator can update an IDVP via the {{ product_name }} Console.
+
+To edit an IDVP:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Find the IDVP and click **Setup up**.
```
Comment (pavinduLakshan) [inline en/includes/guides/identity-verification/manage-identity-verification-provider.md:22]:
```suggestion
2. Find the IDVP and click **Setup**.
```

File: en/includes/guides/identity-verification/manage-identity-verification-provider.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Disable an Identity Verification Provider
+
+An administrator can disable/enable an IDVP via the {{ product_name }} Console.
+
+To disable an IDVP:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Find the IDVP and click **Setup up**.
```
Comment (pavinduLakshan) [inline en/includes/guides/identity-verification/manage-identity-verification-provider.md:37]:
```suggestion
2. Find the IDVP and click **Setup**.
```

File: en/includes/guides/identity-verification/manage-identity-verification-provider.md (hunk 2 above)
Comment (JayaShakthi97) [reply en/includes/guides/identity-verification/manage-identity-verification-provider.md:22]: In the UI, this is mentioned as `Setup up`. Therefore keeping as it is in the documentation.

File: en/includes/guides/identity-verification/manage-identity-verification-provider.md (hunk 3 above)
Comment (JayaShakthi97) [reply en/includes/guides/identity-verification/manage-identity-verification-provider.md:37]: Refer to: https://github.com/wso2/docs-is/pull/4803/files#r1792857699

File: en/identity-server/next/docs/apis/restapis/identity-verification-providers.yaml (hunk 1 above)
Comment (JayaShakthi97) [reply en/identity-server/next/docs/apis/restapis/identity-verification-providers.yaml]: Fixed: f31e6771411756676f904556a8900e19e6acce6a

File: en/includes/guides/identity-verification/manage-identity-verification-provider.md (hunk 2 above)
Comment (JayaShakthi97) [reply en/includes/guides/identity-verification/manage-identity-verification-provider.md]: In the UI, this is mentioned as `Set up`.
Updated: f31e6771411756676f904556a8900e19e6acce6a

File: en/includes/guides/identity-verification/manage-identity-verification-provider.md (hunk 3 above)
Comment (JayaShakthi97) [reply en/includes/guides/identity-verification/manage-identity-verification-provider.md]: https://github.com/wso2/docs-is/pull/4803#discussion_r1792863663

**Comment by AfraHussaindeen** [comment]
Add Identity Verification Claim Management APIs as well. Refer https://github.com/wso2/identity-api-user/blob/master/components/org.wso2.carbon.identity.api.user.idv/org.wso2.carbon.identity.api.user.idv.v1/src/main/resources/idv.yaml

**Comment by JayaShakthi97** [comment]
> Add Identity Verification Claim Management APIs as well. Refer [wso2/identity-api-user@`master`/components/org.wso2.carbon.identity.api.user.idv/org.wso2.carbon.identity.api.user.idv.v1/src/main/resources/idv.yaml](https://github.com/wso2/identity-api-user/blob/master/components/org.wso2.carbon.identity.api.user.idv/org.wso2.carbon.identity.api.user.idv.v1/src/main/resources/idv.yaml?rgh-link-date=2024-10-10T04%3A39%3A58Z)

Added via 117a297207cb2e7eefcc6c8a02c782217541f3ca

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**